src/libraries/System.Threading.Tasks.Dataflow/src/Internal/AsyncEnumerable.cs

```diff
@@ -0,0 +1,80 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
Delete this whole file.  We must not block like this.

Reply:
Indeed!